portfolio/src/main/webapp/index.html

```diff
@@ -17,22 +17,35 @@ <h1>Sarah Adebabay's Portfolio</h1>
       <button onclick="addRandomGreeting()">Fun Fact</button>
       <div id="greeting-container"></div>
     </div>
-    
     <div id="content">
       <h1>Let's Connect</h1>
       <h2>Github</h2>
       <a href="https://github.com/sarah-adebabay">https://github.com/sarah-adebabay</a>
       <h2>LinkedIn</h2>
       <a href="linkedin.com/in/sarah-adebabay">linkedin.com/in/sarah-adebabay</a>
+      <h2>Hello</h2>
+      <p>Click here for a fun surprise!</p>
+      <button onclick="getName()">Get it!</button>
+      <div id="name-container"></div>
+      <button onclick="getComments()">Comment</button>
+      <div id="comments-container"></div>
     </div>
 
+    <form action="/data" method="POST">
+      <p>Leave a comment:</p>
+      <input type="text" name="comment-left">
+      <br/><br/>
+      <input type="submit" value="Submit">
+    </form>
+  </body>
+
     <div id="content">
         <h1>Photo Gallery</h1>
         <p>Here is a gallery of photographs that I took on my travels!</p>
     </div>
 
     <div id="content">
-        <a target="_blank" href="images/IMG_8982.jpg">
+        <a target="_blank" href="/_ah/admin.">
         <img src="images/IMG_8982.jpg" width="650"></a>
```

Comment:
You probably don't want to expose this link in your webpage since the majority of people won't (and shouldn't) have permissions to view this page since they're not admins of your Google DataStore.